cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/utils.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Is the functionality in this utils file already available in the existing codebase? If so, I'd be keen to import and re-use that?

Reply:
yeah, kind of, but its private to asset-syncer. now. Needs to be re-packaged for re-used. I will put a TODO item to see if I can clean it up